src/sensor/settings.py

```diff
@@ -366,6 +367,7 @@
         "tasks": {"handlers": ["console"], "level": LOGLEVEL},
         "scos_actions": {"handlers": ["console"], "level": LOGLEVEL},
         "scos_usrp": {"handlers": ["console"], "level": LOGLEVEL},
+        "scos_tekrsa": {"handlers": ["console"], "level": LOGLEVEL},
         "scos_sensor_keysight": {"handlers": ["console"], "level": LOGLEVEL},
         "scos_tekrsa": {"handlers": ["console"], "level": LOGLEVEL},
```

Comment:
`scos_tekrsa` is included twice here. Also `scos_usrp` is still included (although this is probably OK)

Reply:
Removed new/redundant entry in [027c83e](https://github.com/NTIA/scos-sensor/pull/218/commits/027c83edd17c1ef51e5d4c1a7d100cf2617e3c81). Leaving `scos_usrp` and `scos_sensor_keysight` is fine here.